src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ObjectionService.java

```diff
@@ -219,14 +219,14 @@ private void deleteFromS3(String requestId, String attachmentId, Map<String, Obj
                 String message = String.format(ATTACHMENT_NOT_DELETED_SHORT, attachmentId);
                 logger.infoContext(requestId, message, logMap);
                 throw new ServiceException(message);
-            } else {
-                if (response.getHttpStatus().isError()) {
-                    String message = String.format(ATTACHMENT_NOT_DELETED,
-                            attachmentId, response.getHttpStatus());
-                    logger.infoContext(requestId, message, logMap);
-                    throw new ServiceException(message);
-                }
             }
+            if (response.getHttpStatus().isError()) {
```

Comment:
just formatting, reducing levels of nesting (sonar complexity rating)

Reply:
These could probably go into a separate method.

Reply:
tried having a separate method but it didn't feel right checking the response from the call in a different method. I've refactored it, see if you think it is any better. if not I can revert it back to the original.